pom.xml

```diff
@@ -47,14 +52,24 @@
     <scm>
         <connection>scm:git:git://github.com/jenkinsci/msbuild-plugin.git</connection>
         <developerConnection>scm:git:git@github.com:jenkinsci/msbuild-plugin.git</developerConnection>
-        <tag>msbuild-1.26</tag>
+        <tag>${scmTag}</tag>
     </scm>
 
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>io.jenkins.tools.bom</groupId>
+                <artifactId>bom-2.222.x</artifactId>
+                <version>24</version>
+                <scope>import</scope>
+                <type>pom</type>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
```

Comment:
given you only have 1 dependency here - do you really want this?

Reply:
Yes, regardless of number of dependencies, using the bom will avoid future PCT version mismatch failures.

Reply:
can you explain that - as the BOM is not a plugin dependency and not shipped with a war so how is it updated and how will it help here?
https://github.com/jenkinsci/plugin-compat-tester/blob/a7aacd8b4907aed68248c06af82eb631e4e10369/plugins-compat-tester/src/test/resources/org/jenkins/tools/test/model/credentials-binding-pom-after.xml#L48-L56 and https://github.com/jenkinsci/plugin-compat-tester/blob/a7aacd8b4907aed68248c06af82eb631e4e10369/plugins-compat-tester/src/test/resources/org/jenkins/tools/test/model/credentials-binding-pom-before.xml#L54-L62 show the same version so some pointers so I can understand this would be very helpful

Reply:
@jtnord 

Thought they were functionally different in some scenarios for enforcement of minimum upper bounds, but I was probably wrong.  The last three commits in https://github.com/bitwiseman/github-branch-source-plugin/commits/dependencies/sample-1 show that the error is the same regardless of where the version is set.

Putting that aside, I would say that using the bom is considered best practice for all plugins we maintain.  If someone comes along later and adds another dependency, the bom will already be there.  Using the bom is more lines in this case, but it feels clearer.  

Also, `dependencyManagement` is the best practice for ensuring specific version numbers for transitive dependencies (rather than depending on pom file and class loading ordering). I see the point that there is only one dependency, so this doesn't apply here, but again start with consistent best practices and stick with them.

Just my opinion.